securedrop_client/gui/widgets.py

```diff
@@ -667,15 +667,15 @@ def __init__(self, source_db_object: Source, sdc_home: str, controller: Client,
 
         self.update_conversation(self.source.collection)
 
+    def clear_conversation(self):
+        while self.conversation_layout.count():
+            child = self.conversation_layout.takeAt(0)
+            if child.widget():
```

Comment:
Since `takeAt(0)` could return `None`, should be `if child is not None and child.widget()`?

Reply:
Since the while loop now checks that `self.conversation_layout.count()` is not zero, it's not necessary to check whether or not `takeAt(0)` is `None` anymore.

Reply:
Oh right derp.